modules/oe_theme_content_project/src/Plugin/ExtraField/Display/FundingExtraField.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_content_project\Plugin\ExtraField\Display;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\extra_field\Plugin\ExtraFieldDisplayFormattedBase;
+
+/**
+ * Extra field displaying Funding and Proposals Project fields.
+ *
+ * @ExtraFieldDisplay(
+ *   id = "oe_theme_content_project_funding",
+ *   label = @Translation("Funding"),
+ *   bundles = {
+ *     "node.oe_project",
+ *   },
+ *   visible = true
+ * )
+ */
+class FundingExtraField extends ExtraFieldDisplayFormattedBase {
+
+  use StringTranslationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function viewElements(ContentEntityInterface $entity) {
+    $funding = $entity->get('oe_project_funding_programme')->getIterator();
+    foreach ($funding as $item) {
+      $build['#items'][] = [
+        'fieldlabel' => $entity->oe_project_funding_programme->getFieldDefinition()->getLabel(),
```

Comment:
Are we sure that the field label there gets correctly translatable when displaying this in a language other than English?

Reply:
Also, please always separate different words in a name, like `field_label`.

Reply:
It shows translated labels:
<img width="1035" alt="labels" src="https://user-images.githubusercontent.com/5982573/86921307-3975d700-c12b-11ea-865d-a46447e044c9.png">